.travis.yml

```diff
@@ -7,6 +7,5 @@ install:
   - sudo pip install -r requirements.txt
 
 script:
-  - sudo python -m pytest . --ignore=rhit-xprize-neural-network
   - cd server
-  - pylint -E *.py
```

Comment:
Did you ever figure this out? I'm fine leaving it out for now, but it'd be nice to know what's going on.

Reply:
This is brought up in #44.

Reply:
@kgroble I didn't ever figure it out. I think that travis is just trying very hard to make my life unpleasant.